src/libZilliqa/Zilliqa.cpp

```diff
@@ -148,6 +148,41 @@ Zilliqa::Zilliqa(const PairOfKey& key, const Peer& peer, SyncType syncType,
     LOG_STATE("[IDENT] " << string(key.second).substr(0, 8));
   }
 
+  // Logging useful information
+  std::string startupInfo{"Node startup Information\n"};
+
+  // 3 mode: IP/Key/P2P Seed Key (ENABLE_SEED_TO_SEED_COMMUNICATION)
+  startupInfo += "Whitelisting mode \t= ";
+  startupInfo += multiplierSyncMode
+                     ? ENABLE_SEED_TO_SEED_COMMUNICATION
+                           ? "Public Key (P2P Seed) whitelisting mode\n"
+                           : "Public Key only whitelisting mode\n"
+                     : "IP whitelisting mode\n";
+
+  // IP Address
+  startupInfo += "IP Address \t\t= ";
+  startupInfo += peer.GetPrintableIPAddress();
+  startupInfo += "\n";
+
+  // Public Key
+  bytes publicKey;
```

Comment:
Is there any other ways to get the public Key?

Also this is also being log in https://github.com/Zilliqa/Zilliqa/blob/b689bbdfe716b16110fe05e87bdbcbeea39a0bd5/src/libZilliqa/Zilliqa.cpp#L41

Reply:
`extSeedKey` is the key which we intend to display here.`extSeedKey` is different from the node's own keys. This you can say, is an extra pair of keys use for whitelisting to identify a trused valid node.
Display `extSeedKey` here instead.